File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):  
    max_w = 0
    max_h = 0
    
    for size in sizes:
        size.sort(reverse = True)
        max_w = max(max_w, size[0])
        max_h = max(max_h, size[1])
        
    
    return max_w * max_h

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let max_w = 0, max_h = 0;
    
    for (let size of sizes) {
        sort_size = size.sort((a, b) => a - b);
        max_w = Math.max(max_w, sort_size[0]);
        max_h = Math.max(max_h, sort_size[1]);
    }
        
    return max_w * max_h;
}